extensions/wikia/ScavengerHunt/tests/ScavengerHuntTest.php

```diff
@@ -1,5 +1,5 @@
 <?php
-class ScavengerHuntTest extends WikiaBaseTest {
+abstract class ScavengerHuntTest extends WikiaBaseTest {
```

Comment:
Yeah. All tests were marked as `Broken` 4 years ago and the extension might be disabled now.